<ui version="4.0" >
 <class>Ui_gpsTrackerOptions</class>
 <widget class="QDialog" name="Ui_gpsTrackerOptions" >
  <property name="windowModality" >
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Options For GPS Tracker</string>
  </property>
  <property name="windowIcon" >
   <iconset>
    <normaloff>../../../../.designer/backup/Satellite-22.png</normaloff>../../../../.designer/backup/Satellite-22.png</iconset>
  </property>
  <property name="styleSheet" >
   <string notr="true" >font: 11pt "Arial";</string>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <widget class="QGroupBox" name="groupMarker" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>210</y>
     <width>371</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title" >
    <string>Marker Options</string>
   </property>
   <widget class="QLabel" name="label" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>30</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Marker Type:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbxMarkerType" >
    <property name="geometry" >
     <rect>
      <x>120</x>
      <y>20</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip" >
     <string>Select the type of marker to display the position</string>
    </property>
    <property name="statusTip" >
     <string>Select the type of marker to display the position</string>
    </property>
    <property name="maxCount" >
     <number>1</number>
    </property>
    <property name="minimumContentsLength" >
     <number>1</number>
    </property>
    <property name="modelColumn" >
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>80</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Marker Fill Colour</string>
    </property>
   </widget>
   <widget class="QToolButton" name="btnMarkerFillColor" >
    <property name="geometry" >
     <rect>
      <x>130</x>
      <y>70</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="palette" >
     <palette>
      <active>
       <colorrole role="Button" >
        <brush brushstyle="SolidPattern" >
         <color alpha="255" >
          <red>240</red>
          <green>240</green>
          <blue>240</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base" >
        <brush brushstyle="SolidPattern" >
         <color alpha="255" >
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window" >
        <brush brushstyle="SolidPattern" >
         <color alpha="255" >
          <red>240</red>
          <green>240</green>
          <blue>240</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Button" >
        <brush brushstyle="SolidPattern" >
         <color alpha="255" >
          <red>240</red>
          <green>240</green>
          <blue>240</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base" >
        <brush brushstyle="SolidPattern" >
         <color alpha="255" >
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window" >
        <brush brushstyle="SolidPattern" >
         <color alpha="255" >
          <red>240</red>
          <green>240</green>
          <blue>240</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Button" >
        <brush brushstyle="SolidPattern" >
         <color alpha="255" >
          <red>240</red>
          <green>240</green>
          <blue>240</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base" >
        <brush brushstyle="SolidPattern" >
         <color alpha="255" >
          <red>240</red>
          <green>240</green>
          <blue>240</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window" >
        <brush brushstyle="SolidPattern" >
         <color alpha="255" >
          <red>240</red>
          <green>240</green>
          <blue>240</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="text" >
     <string>...</string>
    </property>
    <property name="toolButtonStyle" >
     <enum>Qt::ToolButtonTextOnly</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3" >
    <property name="geometry" >
     <rect>
      <x>170</x>
      <y>80</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Marker Outline Colour</string>
    </property>
   </widget>
   <widget class="QToolButton" name="btnMarkerOutlineColor" >
    <property name="geometry" >
     <rect>
      <x>320</x>
      <y>70</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text" >
     <string>...</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupMarker_2" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>340</y>
     <width>371</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title" >
    <string>Track Line Options</string>
   </property>
   <widget class="QLabel" name="label_4" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>30</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Track Width:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5" >
    <property name="geometry" >
     <rect>
      <x>160</x>
      <y>30</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Track Line Colour</string>
    </property>
   </widget>
   <widget class="QToolButton" name="btnTrackLineColor" >
    <property name="geometry" >
     <rect>
      <x>290</x>
      <y>20</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text" >
     <string>...</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sbxTrackWidth" >
    <property name="geometry" >
     <rect>
      <x>100</x>
      <y>30</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum" >
     <number>1</number>
    </property>
    <property name="maximum" >
     <number>20</number>
    </property>
   </widget>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox" >
   <property name="geometry" >
    <rect>
     <x>50</x>
     <y>450</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons" >
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbxSaveGPSTrack" >
   <property name="geometry" >
    <rect>
     <x>130</x>
     <y>420</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>Save GPS track to a SHAPE file</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupGPSSettings" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>10</y>
     <width>351</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title" >
    <string>Select GPS Connection Settings</string>
   </property>
   <widget class="QRadioButton" name="rbtTryAll" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>30</y>
      <width>191</width>
      <height>18</height>
     </rect>
    </property>
    <property name="toolTip" >
     <string>Set this to try all ports and all speeds</string>
    </property>
    <property name="statusTip" >
     <string>Set this to try all ports and all speeds</string>
    </property>
    <property name="whatsThis" >
     <string>Forces the searching for the connection settings</string>
    </property>
    <property name="text" >
     <string>Search for a Connection</string>
    </property>
    <property name="checked" >
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbtSetConnection" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>80</y>
      <width>211</width>
      <height>18</height>
     </rect>
    </property>
    <property name="toolTip" >
     <string>Select a serial port and port speed</string>
    </property>
    <property name="statusTip" >
     <string>Select a serial port and port speed</string>
    </property>
    <property name="text" >
     <string>Select Connection Settings</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6" >
    <property name="geometry" >
     <rect>
      <x>70</x>
      <y>60</y>
      <width>46</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text" >
     <string>&lt; OR ></string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>110</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Port Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8" >
    <property name="geometry" >
     <rect>
      <x>190</x>
      <y>110</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Port Speed</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbxPortName" >
    <property name="enabled" >
     <bool>false</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>140</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="toolTip" >
     <string>Select the serial port to be connected</string>
    </property>
    <property name="statusTip" >
     <string>Select the serial port to be connected</string>
    </property>
    <property name="whatsThis" >
     <string>Name of serial port</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbxPortSpeed" >
    <property name="enabled" >
     <bool>false</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>190</x>
      <y>140</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="toolTip" >
     <string>Select the baud rate of the serial port to be connected</string>
    </property>
    <property name="statusTip" >
     <string>Select the baud rate of the serial port to be connected</string>
    </property>
    <property name="whatsThis" >
     <string>Baud Rate of serial port</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Ui_gpsTrackerOptions</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>220</x>
     <y>465</y>
    </hint>
    <hint type="destinationlabel" >
     <x>199</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Ui_gpsTrackerOptions</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>220</x>
     <y>465</y>
    </hint>
    <hint type="destinationlabel" >
     <x>199</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
